src/web/theme/pages/TryIt/TryIt.js

```diff
@@ -40,9 +42,45 @@ const steps = [
       </Fragment>
     )
   },
-  { name: "Why", renderStep: () => <div>Work In Progress</div> }
+  {
+    name: "Why",
+    renderStep: props => (
+      <Fragment>
+        <SelectStore
```

Comment:
Now that I see the complete implementation, I think that it would have been better to structure this differently, as there is a lot of needless duplication.

In my opinion, you should only have to use each component once. This would once again imply a fairly big refactoring though :(